packages/backend/src/tests/propertyBased/tablesToSkip.ts

```diff
@@ -0,0 +1,9 @@
+export const tablesToSkip = [
```

Comment:
Add a comment explaining why we skip these. We will not remember 1 year from today.